### PR TITLE
[REVIEW] refactor dns extractor and ip

### PR DESCRIPTION
**Pull request by bsuryadevara**
- Improved memory utilization and performance by removing unwanted columns before merge operation. Add delete operation on variables to avoid data duplication in the memory.

### CODE REVIEW

File: clx/dns/dns_extractor.py [hunk 1]
```diff
@@ -309,25 +305,27 @@ def parse_url(url_df_col, req_cols=None):
     sv = DnsVarsProvider.get_instance()
     req_cols = _verify_req_cols(req_cols, sv.allowed_output_cols)
     col_dict = _create_col_dict(req_cols, sv.allowed_output_cols)
-    hostnames = extract_hostnames(url_df_col)
+    hostnames = extract_hostnames(url_series)
+    url_index = url_series.index
+    del url_series
```
Comment (BartleyR) [inline clx/dns/dns_extractor.py]: This is a good practice.

**Comment by brhodes10** [comment]
FYI @bsuryadevara there are merge conflicts

**Comment by bsuryadevara** [comment]
> FYI @bsuryadevara there are merge conflicts

resolved conflicts. thanks for the findings.

**Comment by brhodes10** [comment]
rerun tests

**Comment by brhodes10** [comment]
rerun tests

File: ci/gpu/build.sh [hunk 2]
```diff
@@ -47,7 +47,7 @@ conda config --set ssl_verify False
 conda install nvstrings=${MINOR_VERSION} cugraph=${MINOR_VERSION} dask-cudf=${MINOR_VERSION} \
    requests yaml python-confluent-kafka python-whois dask
 
-conda install -y pytorch torchvision -c pytorch
+conda install -y pytorch==1.3.1 torchvision -c pytorch
```
Comment (dillon-cullinan) [inline ci/gpu/build.sh]: This should also be reflected in conda/recipes/meta.yml

**Pull request merged**